tools/capture_har/capturer/index.js

```diff
@@ -61,29 +61,32 @@ class HarCapturer {
       
       if(this.options.extractCriticalRequests) {
         await client.Runtime.enable();
+        console.log("enabled runtime")
```

Comment:
change all `console.log` to `console.error` if not used in the final output (otherwise this will mess with using the script and piping its output where someone would expect just the HAR)